### PR TITLE
update various unmaintained Python ports to their latest version

### PR DESCRIPTION
**Pull request by reneeotten**
#### Description
- py-chm: fix livecheck, add license, modernize checksums 
- py-cjson: update to 1.2.1, enable tests, default livecheck, update homepage, install files in post-destroot
- py-clientform: update to 0.2.10, switch to PyPI
- py-clonedigger: update to 1.1.0, fix dependency type, use default PyPI livecheck
- py-cog: update to 2.5.1, default PyPI livecheck
- py-config: fix livecheck, modernize checksums

###### Tested on

macOS 10.14.3 18D42
Xcode 10.1 10B61

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? N/A
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`? Yes, where applicable.
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-chm/Portfile [hunk 1]
```diff
@@ -9,14 +9,18 @@ revision            1
 categories-append   textproc
 platforms           darwin
 maintainers         nomaintainer
+license             GPL
```
Comment (mf2k) [inline python/py-chm/Portfile]: We like to be more specific about which version(s) of the GPL is actually used. Some examples are GPL-2, GPL-2+, etc.

Comment (mf2k) [reply python/py-chm/Portfile]: Looking at the homepage, it should be `GPL-2+`.

Comment (reneeotten) [reply python/py-chm/Portfile]: thanks Frank! You're right about the GPL-2+; I have changed it accordingly.

**Comment by mf2k** [comment]
Thank you!

**Pull request merged**